### ISSUE TITLE
Capture metrics to track user behavior

### ISSUE THREAD

**Issue by jaclynperrone** [hammerlab/cycledash#712]
It'll be great to put in some Google Analytics to track clicks and page views throughout CycleDash.

**Comment by ihodes** [comment]
Definitely. I'd vote for a solution that doesn't tie us to a single metric aggregator, though.

**Comment by jaclynperrone** [comment]
You mean use other platforms in addition to Google?

**Comment by ihodes** [comment]
Ideally, I'd like to avoid hardcoding Google Analytics or any single metrics application calls directly into the application.

**Comment by jaclynperrone** [comment]
Ah yes. +1
